javascript/stimulus_reflex.js

```diff
@@ -176,7 +172,7 @@ const extendStimulusController = controller => {
     //
     stimulate () {
       clearTimeout(controller.StimulusReflex.timeout)
-      const url = window.location.href
+      const url = location.href
```

Comment:
According to ESLint, it's actually supposed to be window.location.href.

I honestly don't care either way. :smiling_imp:

Reply:
IMO that’s an artifact of JS culture’s disdain for globals. I see it as bike shedding similar to the semi-colon debate, but I honestly don’t care either.